### DISCUSSION TITLE
Add  ssl_ciphers

### DISCUSSION THREAD

**Discussion by OpenK54** [ZoeyVid/NPMplus#281] [Q&A] [answered]
When I do check with ssl checker it reports that cipher order is not configured which is considert an insecure behavior since clients may not send the more secure cipher suites first. It would be easy to help it by simply add it to the config file (see an example here: https://ssl-config.mozilla.org/). I hope this will be added soon. Otherwise one has to manually ssh into the container and change the config file.

**Comment by Zoey2936** [comment]
what ssl checker do you mean exactly?

**Comment by Zoey2936** [comment]
I already included the Mozilla Intermediate config, but they made a change to the chippers of this config, I pushed this change to develop

**Comment by OpenK54** [comment]
![grafik](https://github.com/ZoeyVid/nginx-proxy-manager/assets/132947170/c9e05730-cecd-4fef-a239-905267736ced)

**Comment by OpenK54** [comment]
https://testtls.com/

**Reply by Zoey2936** [reply]
the problem was not the ciphers themselves, the problem was, that `ssl_prefer_server_ciphers` was set to off (as recommended by mozilla), but because of the strength of ciphers the order should make no problem, but I've enabled it

**Reply by Zoey2936** [reply]
see: https://github.com/mozilla/server-side-tls/issues/260

**Reply by Zoey2936** [reply]
I'm not sure if it would be better to keep it off...

**Reply by OpenK54** [reply]
Me neither. Just saw the result of the check and figured there must be something wrong. All the better if not.

**Reply by Zoey2936** [reply]
If it is on, security should be better, but in a way that is better, but doesn't improve it noticeable, if it off the client compatibility should be better

**Reply by OpenK54** [reply]
I'd go with security.:-) Thanks!

**Reply by Zoey2936** [reply, marked as answer]
I've enabled it and pushed it now to latest.

**Comment by OpenK54** [comment]
Good to know that you allready pushed this change to develop so the ciphers will be included. Thanks for all your effort!
